bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/zk/ZKMetadataDriverBase.java

```diff
@@ -120,23 +125,27 @@ protected void initialize(AbstractConfiguration<?> conf,
                               Optional<Object> optionalCtx) throws MetadataException {
         this.conf = conf;
 
-        String metadataServiceUriStr;
+        final String metadataServiceUriStr;
```

Comment:
why not construct metadataServiceUri directly?

Reply:
any issues you are seeing not construct `metadataServiceUri`? I am not sure what to fix here?

Reply:
what I'm saying is that metadataServiceUriStr isn't needed. It's used on one other place, and metadataServiceUri could equally be used there directly.

Reply:
Sure.